### ISSUE TITLE
observer gets called for a new type

### ISSUE THREAD

**Issue by calibr** [y-js/yjs#155]
If a type is created and changed in the same transaction its observer gets called. Code example:

```
    const map = this.y.get('testmap', YMap)
    transact(() => {
      const newType = new YText()
      newType.observe(function() {
        console.log('CHANGED') // will be printed
      })
      map.set('item1', newType)
      // if the next line is removed the observer won't be called
      newType.applyDelta([{  insert: 'text'  }])
    })
```

I am not sure if this is expected behaviour, but in version 13.0.0-78 observers won't fire for new types even if they gets changed in the same transaction.

**Comment by dmonad** [comment]
I remember that I fixed this issue. Can you test this again with `v13.0.0-90`?

**Comment by calibr** [comment]
still can reproduce this issue with `13.0.0-90` with the code above

**Comment by dmonad** [comment]
Thanks for reporting this. You are absolutely right about this. 

The fix will be published as `13.0.0-91` in a few minutes.

**Comment by calibr** [comment]
@dmonad thank you for fixing this, will check it out
